XenAdmin/Controls/CustomDataGraph/DataSet.cs

```diff
@@ -285,6 +285,21 @@ public static DataSet Create(string uuid, IXenObject xo, bool show, string setty
                 }
                 dataSet.Type = DataType.Gpu;
             }
+            else if (settype.StartsWith("pvscache"))
+            {
+                if (settype.Contains("traffic") || settype.EndsWith("evicted"))
+                    dataSet.CustomYRange = new DataRange(1, 0, 1, Unit.BytesPerSecond, RangeScaleMode.Auto);
+                else if (settype.EndsWith("read_total") || settype.EndsWith("read_hits") || settype.EndsWith("read_misses"))
+                    dataSet.CustomYRange = new DataRange(1, 0, 1, Unit.CountsPerSecond, RangeScaleMode.Auto);
+                else if (settype.Contains("utilization"))
+                {
+                    dataSet.CustomYRange = new DataRange(100, 0, 10, Unit.Percentage, RangeScaleMode.Fixed);
+                    dataSet.MultiplyingFactor = 100;

```

Comment:
As discussed, the multiplying factor should be 1, because the datasource's values range from 0 to 100 (unlike other similar datasources which are 'fractions' ranging from 0 to 1)